build.js

```diff
@@ -143,7 +142,7 @@ function buildlocale (source, locale) {
     .use(layouts({
       engine: 'handlebars',
       pattern: '**/*.html',
-      partials: mapHandlebarsPartials(metalsmith, 'layouts', 'partials'),
+      partials: 'layouts/partials',
```

Comment:
Rest of the script uses `path.join()` so we should stick to that.

@woodyrew Can you add this small change, please? Then it's good to merge.